docs/buildah-umount.md

```diff
@@ -9,11 +9,23 @@ buildah\-umount - Unmount the root file system on the specified working containe
 ## DESCRIPTION
 Unmounts the root file system on the specified working containers.
 
+## OPTIONS
+**--all, -a**
+
+All of the currently mounted containers will be unmounted.
+
 ## EXAMPLE
 
 buildah umount containerID
 
 buildah umount containerID1 containerID2 containerID3
 
+buildah umount --all
+
+43e674866c4cb46dfc64988b95a8247d6ae5fd17f0b15a46f16a81cb62a55353
```

Comment:
We don't have example output from the other commands here and I'm not sure it adds much to have it  here.  I'd drop it, but if we are going to keep it then we ought to add it to the other examples too.

Reply:
updated, I'm ok whether add or not.